packages/brand-moment/src/stories/BrandMoment.stories.tsx

```diff
@@ -0,0 +1,129 @@
+  component: BrandMoment,
+  parameters: {
+    info: {
+      text: `
+        import { BrandMoment } from "@kaizen/brand-moment";
+      `,
+    },
+  },
+  decorators: [story => <div style={{ margin: "-1rem" }}>{story()}</div>],
+}
+
+export const DemoIntro = () => (
+  <BrandMoment
+    backgroundColor="blue"
+    header={<MinimalBasic />}
+    illustration={<EmptyStatesAction alt="" />}
+    subheading="Welcome to Culture Amp"
+    heading="Let's dive in and see how it works"
+    primaryAction={{
+      label: "Get started",
+      href: "#",
+      icon: arrowRightIcon,
+      iconPosition: "end",
+    }}
+  />
+)
+DemoIntro.storyName = "Demo Intro"
+
+export const CaptureIntro = () => (
+  <BrandMoment
+    backgroundColor="blue"
+    header={<MinimalCustomerFocused />}
+    illustration={<EmptyStatesPositive alt="" />}
+    subheading="A survey for Hooli"
+    heading="Manager Effectiveness Survey"
+    body="Thank you for taking the time to respond to this survey. It'll help us better understand your experience and perspective."
+    primaryAction={{
+      label: "Take survey",
+      href: "#",
+      icon: arrowRightIcon,
+      iconPosition: "end",
+    }}
+    secondaryAction={{
+      label: "About data safety",
+      icon: securityTipIcon,
+    }}
+    footer="Your responses and information are securely collected and kept by Culture Amp in accordance with our Privacy Policy. Your responses will be reported to Hooli based on the specific rules for this survey. If you have any additional questions, please contact us at support@cultureamp.com."
+  />
+)
+CaptureIntro.storyName = "Capture Intro"
+
+export const CaptureOutro = () => (
+  <BrandMoment
+    backgroundColor="green"
+    header={<MinimalCustomerFocused />}
+    illustration={<EmptyStatesNeutral alt="" />}
+    subheading="Manager Effectiveness Survey"
+    heading="That's it — thank you"
+    body={
+      <>
+        Your responses have been securely recorded. If you need to, you can{" "}
+        <a href="#">retake the survey</a>.
+      </>
+    }
+    primaryAction={{
+      label: "Go to Home",
+      href: "#",
+      icon: arrowRightIcon,
+      iconPosition: "end",
+    }}
+    secondaryAction={{
+      label: "Rate this survey",
+      icon: feedbackClassifyIcon,
+    }}
+    footer="Your responses and information are securely collected and kept by Culture Amp in accordance with our Privacy Policy. Your responses will be reported to Hooli based on the specific rules for this survey. If you have any additional questions, please contact us at support@cultureamp.com."
+  />
+)
+CaptureOutro.storyName = "Capture Outro"
+
+export const Error = () => (
+  <BrandMoment
+    backgroundColor="red"
+    header={<FakeNavBar />}
+    illustration={<EmptyStatesNegative alt="" />}
+    heading="Missing pages are one of life's mysteries"
+    body={
+      <>
+        <Box mb={1.75}>
+          Sorry but we can't find the page you're looking for. Go back and try
```

Comment:
Same with the language here. I reckon we could change this example to be more direct (and in line with our guidelines):

```diff
- Sorry but we can't find the page you're looking for.
+ We can’t find the page you’re looking for.
```

Reply:
I'm cool with this but CC @katsambarcus because I've just taken this from the Brand Moments Figma, which should be updated as well if we're changing this.

Reply:
I'd prefer if @lucy-oldridge made the call on these language changes.

Reply:
Lucy doesn’t have Github access (no more licenses) so responding on her behalf:
> Would prefer to keep it as is. I think it’s a lightweight way to create an apologetic tone (considering the user could well be frustrated or stressed), especially when the H1 is more abstract and brand-y.